### PR TITLE
Sign in view added with dummy data. UI fueled with material components.

### PR DESCRIPTION
**Pull request by pablodri08**
* [Title of the card](https://androidtraining.atlassian.net/browse/ATP-163?atlOrigin=eyJpIjoiMjc3N2I3ZmYzNDI4NDk4Y2JkZjUwOTc2YmZjYTNiOTIiLCJwIjoiaiJ9)

## Description:

SignInView

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -29,6 +29,7 @@ if (getEnvVariable('KEYSTORE_FILE') != null) {
 
 repositories {
     maven { url 'https://maven.fabric.io/public' }
+    google()
```
Comment (mirland) [inline app/build.gradle]: you can remove both, they are added from the versions file

File: app/build.gradle [hunk 2]
```diff
@@ -160,6 +162,7 @@ dependencies {
     implementation deps.retrofit2.gsonAdapter
     implementation deps.stetho
     implementation deps.timber
+    implementation deps.material
```
Comment (mirland) [inline app/build.gradle]: alpha

File: app/src/main/java/com/xmartlabs/taskloans/ui/screens/signin/SignInFragment.kt [hunk 3]
```diff
@@ -15,47 +15,47 @@ import kotlin.time.ExperimentalTime
  * Created by mirland on 25/04/20.
  */
 class SignInFragment : BaseViewBindingFragment<FragmentSigninBinding>() {
-  private val viewModel: SignInFragmentViewModel by viewModel()
+    private val viewModel: SignInFragmentViewModel by viewModel()
```
Comment (mirland) [inline app/src/main/java/com/xmartlabs/taskloans/ui/screens/signin/SignInFragment.kt]: the indentation is wrong, please copy the style files from gong https://github.com/xmartlabs/gong/tree/master/.idea

File: app/src/main/java/com/xmartlabs/taskloans/ui/screens/splash/SplashFragment.kt [hunk 4]
```diff
@@ -10,19 +10,19 @@ import org.koin.androidx.viewmodel.ext.android.viewModel
  */
 // This fragment shouldn't have ui, the splash ui should be declared in the activity style.
 class SplashFragment : BaseFragment() {
-  private val viewModel: SplashFragmentViewModel by viewModel()
+    private val viewModel: SplashFragmentViewModel by viewModel()
```
Comment (mirland) [inline app/src/main/java/com/xmartlabs/taskloans/ui/screens/splash/SplashFragment.kt]: reindent the whole project

File: app/src/main/res/font/cairo.xml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<font-family
+    xmlns:android="http://schemas.android.com/apk/res/android">
+    <font
+        android:font="@font/cairo_bold"
+        />
+    <font
+        android:font="@font/cairo_light"
```
Comment (mirland) [inline app/src/main/res/font/cairo.xml]: don't you have to add the weight?

File: app/src/main/res/layout/fragment_signin.xml [hunk 6]
```diff
@@ -1,71 +1,132 @@
...
+
+    <TextView
+        android:id="@+id/signInCreateAccount"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/create_account"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/signInEnterButton"
+        style="@style/TextLink"
```
Comment (mirland) [inline app/src/main/res/layout/fragment_signin.xml]: I'd use something like TextStyle.Link

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -1,3 +1,8 @@
 <resources>
     <string name="app_name">TaskLoans</string>
+    <string name="no_name">NONAME</string>
```
Comment (mirland) [inline app/src/main/res/values/strings.xml]: I usually add the screen prefix, for example screen_sign_in.no_name

File: app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -1,3 +1,8 @@
...
+    <string name="user">Usuario</string>
+    <string name="password">Contraseña</string>
+    <string name="enter">ENTRAR</string>
```
Comment (mirland) [inline app/src/main/res/values/strings.xml]: you can define all cap in the TV

**Review by mirland — APPROVED** [review]

File: versions.gradle [hunk 9]
```diff
@@ -61,6 +61,7 @@ versions.snapshotPublisher = "2.2.0"
 versions.stetho = "1b2c025a14"
 versions.timber = "4.7.1"
 versions.tooLargeTool = "915ad99e83"
+versions.material = "1.3.0"
```
Comment (mirland) [inline versions.gradle]: material version name is androidXMaterial

**Review by mirland — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/xmartlabs/taskloans/ui/screens/signin/SignInFragment.kt [hunk 10]
```diff
@@ -15,47 +15,47 @@ import kotlin.time.ExperimentalTime
...
-  @OptIn(ExperimentalTime::class)
-  private fun setupViewModelCallbacks() = with(viewModel) {
-    viewModelTime.observeSuccessResult(viewLifecycleOwner) { duration ->
-      viewBinding.viewModelTimeTextView.text = "View model time, ${duration.inSeconds} seconds"
+    @SuppressLint("SetTextI18n")
+    @OptIn(ExperimentalTime::class)
+    private fun setupViewModelCallbacks() = with(viewModel) {
+        /*viewModelTime.observeSuccessResult(viewLifecycleOwner) { duration ->
+          viewBinding.viewModelTimeTextView.text = "View model time, ${duration.inSeconds} seconds"
+        }*/
```
Comment (f7deleon) [inline app/src/main/java/com/xmartlabs/taskloans/ui/screens/signin/SignInFragment.kt]: remove this

**Pull request merged**